stylesheet.css

```diff
@@ -35,6 +35,41 @@ body {
   margin-right: 5%;
 }
 
+.m-navbar {
+  justify-content: space-between;
+  top: 0;
+  position: fixed;
+  height: 430px;
```

Comment:
- [ ] Kindly make sure to add the mobile menu height that covers the entire page as per the Figma design.
- [ ] Kindly follow the correct spacing, font size and font colour for the mobile menu options.

Figma design:

<img width="318" alt="Screenshot 2022-05-09 at 11 41 15 PM" src="https://user-images.githubusercontent.com/70269507/167494487-2255ac8c-0914-49f2-843c-f178f6ace178.png">

Your design:

https://user-images.githubusercontent.com/70269507/167494961-57665894-8f09-44e9-87a3-4d8366a41f34.mov